manifests/a/AirExplorer/AirExplorer/4.5.3/AirExplorer.AirExplorer.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: AirExplorer.AirExplorer
+PackageVersion: 4.5.3
+PackageLocale: en-US
+Publisher: AirExplorer
```

Needs to match the `A`dd or `R`emove `P`rograms registry string
```suggestion
Publisher: http://www.airexplorer.net
```